torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -60,49 +46,168 @@ bool validateDomain(TensorView* tv, TensorDomain* new_td) {
       first_mismatch >= (int)tv->getComputeAtPosition();
 }
 
+// Return the max position in consumer that producer can be inlined to
+// Cannot inline:
+//   Reduction dimensions in producer
+//   Block broadcast dimensions in producer
+//   Vectorized dimensions in producer or consumer
+//   Dimensions derived from root dimensions that exist in both but are
+//   unmappable
 unsigned int getReplayablePosPasC(
     TensorView* producer,
     TensorView* consumer,
-    const ComputeAtRootDomainMap& root_map_) {
+    const ComputeAtRootDomainMap& root_map_,
+    const ThreadPredicateMap& pred_map) {
+  // Grab dimensions in producer and consumer that are mappable to eachother
+  // based on the computeAtRootDomainMap. This will tell us which dimensions
+  // can be inlined based on avoiding trying to inline reduction structures.
   auto mappable_roots =
-      root_map_.getMappableDims(producer->domain(), consumer->domain(), true);
+      root_map_.getMappableDims(producer->domain(), consumer->domain());
+
+  // Check if any consumer dimensions are marked as vectorize as producer can
+  // not be inlined to vectorized dimensions in consumer.
+  auto c_dom = consumer->domain()->domain();
+  auto vector_dim_it =
+      std::find_if(c_dom.begin(), c_dom.end(), [](IterDomain* id) {
+        return id->getParallelType() == ParallelType::Vectorize ||
+            id->getParallelType() == ParallelType::MisalignedVectorize;
```

Comment:
Although I don't expect we would have more vector types, there's `isParallelTypeVectorize(ParallelType)` in type.h.